.github/ISSUE_TEMPLATE/vip.md

```diff
@@ -2,18 +2,6 @@
 name: Vyper Improvement Proposal (VIP)
 about: This is the suggested template for new VIPs.
 ---
-## Preamble
-
-    VIP: <to be assigned>
-    Title: <VIP title>
-    Author: <list of authors' names and optionally, email addresses>
-    Type: <Standard Track | Informational | Meta>
-    Status: Draft
-    Created: <date created on, in ISO 8601 (yyyy-mm-dd) format>
-    Requires (*optional): <VIP number(s)>
-    Replaces (*optional): <VIP number(s)>
```

Comment:
Perhaps we should keep VIP#, Title, Created?

Reply:
Well at least Auther & Created, which is useful to. Title is obvious.

Reply:
VIP # is issue number, Auto generated.
Author is original submitter. Maybe a quick blurg about co-authors or original submitters.
Created is also in the issue metadata.

We don't use this preamble for anything outside of GitHub.

Reply:
Hmm... fair enough. Drop it like it's hot.